python-test/features/steps/control_plane_sink.py

```diff
@@ -0,0 +1,170 @@
+    Remove all sinks starting with 'sink_label_name_prefix' from the orb
+
+    :param context: Behave class that contains contextual information during the running of tests.
+    """
+    token = context.token
+    sinks_list = list_sinks(token)
+    sinks_filtered_list = filter_list_by_parameter_start_with(sinks_list, 'name', sink_label_name_prefix)
+    delete_sinks(token, sinks_filtered_list)
+
+
+# Todo uncomment assert
+def create_new_sink(token, name_label, remote_host, username, password, description=None, tag_key='',
+                    tag_value=None, backend_type="prometheus"):
+    """
+
+    Creates a new sink in Orb control plane
+
+    :param (str) token: used for API authentication
+    :param name_label:  of the sink to be created
+    :param remote_host: base url to send metrics to a dashboard
+    :param username: user that enables access to the dashboard
+    :param password: key that enables access to the dashboard
+    :param (str) description: description of sink
+    :param tag_key: the key of the tag to be added to this sink. Default: ''
+    :param tag_value: the value of the tag to be added to this sink. Default: None
+    :param backend_type: type of backend used to send metrics. Default: prometheus
+    :return: (dict) a dictionary containing the created sink data
+    """
+    response = requests.post(base_orb_url + '/api/v1/sinks',
+                             json={"name": name_label, "description": description, "tags": {tag_key: tag_value},
+                                   "backend": backend_type, "validate_only": False,
+                                   "config": {"remote_host": remote_host, "username": username, "password": password}},
+                             headers={'Content-type': 'application/json', 'Accept': '*/*',
+                                      'Authorization': token})
+    assert_that(response.status_code, equal_to(201),
+                'Request to create sink failed with status=' + str(response.status_code))
+
+    return response.json()
+
+
+def get_sink(token, sink_id):
+    """
+    Gets a sink from Orb control plane
+
+    :param (str) token: used for API authentication
+    :param (str) sink_id: that identifies sink to be fetched
+    :returns: (dict) the fetched sink
+    """
+
+    get_sink_response = requests.get(base_orb_url + '/api/v1/sinks/' + sink_id, headers={'Authorization': token})
+
+    assert_that(get_sink_response.status_code, equal_to(200),
+                'Request to get sink id=' + sink_id + ' failed with status=' + str(get_sink_response.status_code))
+
+    return get_sink_response.json()
+
+
+def validate_sink_state(sink_data, state, sink_id):
+    """
+    Validates if the state of existing sink is as expected
+
+    :param (dict) sink_data: sink data
+    :param (str) state: sink expected state
+    :param sink_id: sink identifier
+    """
+    sink_state = sink_data['state']
+    assert_that(sink_state, equal_to(state), f"Sink {sink_id} state failed")
```

Comment:
Do we really need a separate function for making an assertion?

Reply:
We not neeeeeed, but i preferred this way, because we gonna use this many times and I guess is prettier and easier to change the key or message if we want. 
Did not you like it?

Reply:
Creating a wrapper function for a single assertion is an overkill.
What I'm suggesting is calling
```python
assert_that(sink_data['state'], equal_to(state), f"Sink {sink_id} state is not " + state)
```
rather than
```python
validate_sink_state(sink_data, state, sink_id)
```
even when it's done in multiple places. There is no other logic encapsulated in this function other than the assertion.

Reply:
ok.. understood. I already changed it